Source/MLX/Documentation.docc/examples.md

```diff
@@ -15,6 +15,17 @@ Example software for ``MLX`` and `MLXNN`.  Browse it in GitHub at:
 
 - [https://github.com/ml-explore/mlx-swift-examples](https://github.com/ml-explore/mlx-swift-examples)
 
+## MNISTTrainer
+
+An example that runs on both iOS and macOS that downloads MNIST training
+data and trains an MNIST model.
+
+## LLMEval
+
+An example that runs on both iOS and macOS that downloads a LLM model
+weights and tokenizer configuration from Hugging Face and evaluates
+a prompt in-process.
```

Comment:
What do you mean by "a prompt in-process"? That sounds a bit odd.

Maybe say:

```suggestion
An example that runs on both iOS and macOS that downloads a LLM model
weights and tokenizer configuration from Hugging Face and generates text
from a given prompt.
```

Reply:
I was trying to indicate that it was generated locally, not via a server.  I like your phrasing -- do you think we need to emphasize that it is local?  Or perhaps that is obvious from the context :-)

Reply:
Haha yea well it’s certainly clear to me. You could add something like “An example that runs locally on both…”

But I also think it’s clear enough without it